### DISCUSSION TITLE
Questions about v5.2.0 Field Names & "Relink" plugin

### DISCUSSION THREAD

**Discussion by msameen99** [TiddlyWiki/TiddlyWiki5#5795] [Q&A]
Hello every one ...

The liberation of the field names in v5.2.0 will introduce the possibility to revers how we "implement" the relations between Tiddlers.

So, 
- Instead of what I used to do which is create a field name "**parents**" and list all related tiddler names, now we can created a field name **equal** to a tiddler name and the content of that field will hold the value of "MyParent" (and all the relations names that exist between those 2 tiddlers)
- I can use "has[] & get[] operators" instead of "fieldName[] & contains[] operators" to query the relations in that schema

Now I've the following questions:-
1- Is that design pattern will be native in TW5 or it'll be just a hack?i.e., Is there any restriction while working that way?
2- If we can do it natively (and I hope), the "Relink" plugin ability to rename the "Field Names" will be a key part here. so is there a plan to add this possible to "Relink"?

Thanks in advance

**Comment by AnthonyMuscio** [comment]
I would like to add, can we maximise the possibilities of making use of Unicode Characters in field names as well?

**Reply by msameen99** [reply]
What I understood from Jeremy [announcement](https://groups.google.com/g/tiddlywiki/c/eccIEHZoxsI/m/JpxoQdZrAwAJ) is (if I'm not mistaken) that the Ver 5.2.0 will support Unicode.

> Finally, we can have fields called "⛄️".

**Comment by pmario** [comment]
>Instead of what I used to do which is create a field name "parents" and list all related tiddler names, now we can created a field name equal to a tiddler name and the content of that field will hold the value of "MyParent" (and all the relations names that exist between those 2 tiddlers)

This will cause you a lot of performance problems, because the core doesn't contain any filter optimizations for this type of "relations" yet.
